__tests__/app.test.js

```diff
@@ -204,44 +204,159 @@ describe("POST /api/articles/:article_id/comments", () => {
     const postBody = {
       username: "butter_bridge",
       body: "This is a comment about article",
-      votes: 100
+      votes: 100,
     };
     return request(app)
       .post("/api/articles/1/comments")
       .send(postBody)
       .expect(400)
       .then(({ body: { msg } }) => {
-        expect(msg).toBe("Bad Request")
+        expect(msg).toBe("Bad Request");
+      });
+  });
+  test("400: Should response with Bad Request when Post Body is empty", () => {
+    const postBody = {
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(postBody)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+  test("400: Should response with Bad Request when username does not exists", () => {
+    const postBody = {
+      username: "user_name",
+      body: "This is a comment about article",
+      votes: 100,
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(postBody)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
       });
   });
-  test('404: Should response with Bad Request when Article does not exist ', () => {
+  test("404: Should response with Bad Request when Article does not exist ", () => {
     const postBody = {
       username: "butter_bridge",
-      body: "This is a comment about article"
+      body: "This is a comment about article",
     };
     return request(app)
       .post("/api/articles/999/comments")
       .send(postBody)
       .expect(404)
       .then(({ body: { msg } }) => {
-        expect(msg).toBe("Does Not Found")
+        expect(msg).toBe("Does Not Found");
       });
   });
-  test('400: Should response with Bad Request when article_id has invalid format', () => {
+  test("400: Should response with Bad Request when article_id has invalid format", () => {
     const postBody = {
       username: "butter_bridge",
-      body: "This is a comment about article"
+      body: "This is a comment about article",
     };
     return request(app)
       .post("/api/articles/one/comments")
       .send(postBody)
       .expect(400)
       .then(({ body: { msg } }) => {
-        expect(msg).toBe("Bad Request")
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("201: Should update articles votes and return updated article", () => {
+    const updateBody = {
+      inc_vote: 1,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updateBody)
+      .expect(200)
+      .then(({ body: { updatedArticle } }) => {
+        expect(updatedArticle).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
```

Comment:
We're wanting to check that the patch has worked here. If we're just saying votes should be any Number, then this test will always pass. Instead we should be expecting a specific value of whatever the votes was plus one 👍